tests/unit_tests/test_session.py

```diff
@@ -71,6 +72,9 @@ def test_query_success(self, mock_token, mock_env, variables):
                         "/graphql",
                         {"query": me_query, "variables": variables},
                     )
+                    assert resp.url == URL("https://cratedb.local").with_path(
```

Comment:
You could pass the full url string directly

```suggestion
                    assert resp.url == URL("https://cratedb.local/graphql")
```